scripts/script.js

```diff
@@ -0,0 +1,11 @@
+
+function showInfo(name,breed,fee) {
+  result = (name + ' is a ' + breed + ' and can be adopted for ' + fee + '.');
+  alert(result);
+}
+
+
+function addTotal(fee) {
+  result = ('Your current total is $' + fee);
+  alert(result);
```

Comment:
Nice work using parameters to send data to the function. The key here is to keep update a persistent variable with the total. I would create a variable 'total' outside the function on line 1 : 
`let total = 0;`
then inside this function update it like so:
`total += fee;`
then report the total in alert.